### PR TITLE
FE: list service discovery on UI

### PR DESCRIPTION
**Pull request by faizanjanjua**
## Description
Currently, we are already retrieving saved service dicovery via grapghQL call listActiveDiscovery bit does not shoe on the UI.
Reason, Its a new discovery type so create a new mapper that takes saved service doscovery from grapghQL query and return the service discovery type defined on UI and display it.

## Jira link(s)
- https://opennms.atlassian.net/browse/LOK-2754

## Checklist
* [ ] Follows Lōkahi's [development guidelines.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.
* [ ] Notify devops of changes to the Charts.
* [ ] Notify documentation team of any changes to names of screens or features (affects URLs).

### CODE REVIEW

**Review by Shahbaz-dataq — APPROVED** [review]
LGTM

File: ui/src/store/Views/discoveryStore.ts [hunk 1]
```diff
@@ -45,7 +45,8 @@ export const useDiscoveryStore = defineStore('discoveryStore', {
     validateOnKeyUp: false,
     selectedServiceDiscoveryTarget: undefined,
     isSerivceDiscoveryTargetModalVisible: false,
-    serivceDiscoveryTargetMode: CreateEditMode.None
+    serivceDiscoveryTargetMode: CreateEditMode.None,
```
Comment (synqotik) [inline ui/src/store/Views/discoveryStore.ts:48]: Should be `serviceDiscoveryTargetMode` (fix typo)

File: ui/src/store/Views/discoveryStore.ts [hunk 2]
```diff
@@ -276,13 +286,20 @@ export const useDiscoveryStore = defineStore('discoveryStore', {
           await discoveryMutations.addAzureCreds({ discovery: validatedDiscovery})
         } else if (this.selectedDiscovery.type === DiscoveryType.ServiceDiscovery && validatedDiscovery) {
           const targets = (this.selectedDiscovery.meta as DiscoveryServicesMeta).serviceDiscoveryTargets
-          if (!targets || targets.length == 0) {
+          if (this.ServiceDiscoveryFormMode === CreateEditMode.Create) {
+            if (!targets || targets.length == 0) {
+              this.loading = false
+              return showSnackbar({
+                msg: 'Atleast One Discovery Target is Required!'
```
Comment (synqotik) [inline ui/src/store/Views/discoveryStore.ts:293]: Should be `At least one Discovery Target is required.`

File: ui/src/store/Views/discoveryStore.ts [hunk 3]
```diff
@@ -276,13 +286,20 @@ export const useDiscoveryStore = defineStore('discoveryStore', {
...
+              })
+            } else {
+              await discoveryMutations.createSimpleMonitoredActiveDiscovery({ entity: validatedDiscovery })
+            }
+          } else {
             this.loading = false
             return showSnackbar({
-              msg: 'Atleast One Discovery Target is Required!'
+              msg: 'Update operation for this discovery will be is not available yet!'
```
Comment (synqotik) [inline ui/src/store/Views/discoveryStore.ts:301]: Should be: `Update operation for this discovery is not yet available.`

**Review by synqotik — APPROVED** [review]
Looks good, just a few small typos to fix before merging.

**Pull request merged**